.github/workflows/rc_sync.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow will sync changes in the release candidate branch to master during a
+# feature freeze. The action is run daily and will only create a pull request when
+# there exists a release candidate branch corresponding to the current package version
+# as declared in pennylane/_version.py, that is if the branch "v0.23.0-rc0" exists while
+# the package version is "v0.23.0-dev".
+
+name: Sync rc with master
+
+# Controls when the workflow will run
+on:
+  # Scheduled trigger every weekday at 2:47am UTC
+  schedule:
+  - cron:  '47 2 * * 1-5'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job to sync the master branch with changes from the rc
+  sync:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out the PennyLane repository
+      - uses: actions/checkout@v3
+
+      # Check for the existence of an rc branch for the current PennyLane version
+      - name: Check for rc branch
+        run: |
+          VERSION=$(python setup.py --version)
+          IFS=. read MAJ MIN PAT <<< "${VERSION%.dev0}"
```

Comment:
Does this line produce what we'd expect?

Executing it and hitting `echo $IFS`, there's no output.

Also, executing part of it, I'm not sure we get expected outputs here:
```shell
(base) antal@pc:~/xanadu/pennylane$ read MAJ MIN PAT <<< "${VERSION%.dev0}"
(base) antal@pc:~/xanadu/pennylane$ echo $MAJ
0.23.0
(base) antal@pc:~/xanadu/pennylane$ echo $MIN

(base) antal@pc:~/xanadu/pennylane$ echo $PAT
```
Maybe I'm doing something wrong though.